lib/pull_request.rb

```diff
@@ -125,14 +129,31 @@ def tell_dependency_manager_what_dependencies_are_allowed
   end
 
   def tell_dependency_manager_what_dependabot_is_changing
+    dependency_updates = commit_message.scan(/Updates `(\w+)` from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d)/)
+
+    # Commit messages can have different formats
+    if dependency_updates.empty?
+      dependency_updates = commit_message.scan(/Bump (?:\[.*?\]\(.+?\) )?(\w+) from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d)/)
+    end
```

Comment:
I think we could make this more self-documenting, now that we've clarified the two different commit message structures. Something like:

```ruby
single_dependency_update_regex = /Bump (\w+) from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d+)/
multiple_dependency_updates_regex = /Updates `(\w+)` from (\d+\.\d+\.\d+) to (\d+\.\d+\.\d+)/

dependency_updates = commit_message.scan(single_dependency_update_regex)
dependency_updates = commit_message.scan(multiple_dependency_updates_regex) if dependency_updates.empty?
```